src/nsrt_learning/option_learning.py

```diff
@@ -329,9 +331,11 @@ def _regressor_based_policy(self, state: State, memory: Dict,
         if self._is_parameterized:
             # Compute the updated relative goal.
             assert np.allclose(params, memory["params"])
-            changing_objs = [objects[i] for i in self._changing_parameter_idxs]
-            relative_goal_vec = memory["absolute_params"] - state.vec(
-                changing_objs)
+            var_to_obj = dict(zip(self._operator.parameters, objects))
+            state_params = _create_absolute_option_param(
+                state, self._changing_var_to_feat, self._changing_var_order,
```

Comment:
the state here can and will be different than the above one in `initiable()` right? (it'll match for the first timestep, but then there will be many more after that)

Reply:
right